cmd/kpod/stop.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"github.com/containers/storage/pkg/reexec"
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	"os"
+)
+
+const crioConfigPath = "/etc/crio/crio.conf"
+
+var stopTimeout int64
+var configPath string
+
+var (
+	stopFlags = []cli.Flag{
+		cli.Int64Flag{
+			Name:        "timeout, t",
+			Usage:       "Seconds to wait for stop before killing it (default 10)",
+			Value:       10,
+			Destination: &stopTimeout,
+		},
+		cli.StringFlag{
+			Name:		"config",
+			Usage:		"path to configuration file",
+			Value: 		crioConfigPath,
+			Destination: &configPath,
+		},
+	}
+
+	stopDescription = "Stops one or more containers"
+	stopCommand     = cli.Command{
+		Name:        "stop",
+		Usage:       "Stop one or more containers",
+		Description: stopDescription,
+		Flags:       stopFlags,
+		Action:      stopCmd,
+		ArgsUsage:   "CONTAINER-ID [CONTAINER-ID...]",
+	}
+)
+
+func stopCmd(c *cli.Context) error {
+	var hasError bool = true
+	args := c.Args()
+	if len(args) < 1 {
+		return errors.Errorf("specify one or more container IDs")
+	}
+	config := new(server.Config)
+	if err := config.FromFile(configPath); err != nil {
+		if c.GlobalIsSet("config") || !os.IsNotExist(err) {
```

Comment:
Might want to unconditionally return err here. If we error out loading the config, I don't believe it's possible to proceed.